src/syft/core/node/vm/vm.py

```diff
@@ -47,6 +47,9 @@ def __init__(
             verify_key=verify_key,
         )
 
+        # specific location with name
+        self.vm = SpecificLocation(name=self.name)
```

Comment:
same comment as DEvice... not sure this is the right attr name for this attribute

Reply:
mostly because a node can subscribe to many locations

Reply:
Isn't this the implicit one created when a Node is created for itself?
```
bob_vm = sy.VirtualMachine(name="Bob's VM")
assert bob_vm.name == "Bob's VM"
assert bob_vm.vm.name == "Bob's VM"
```